### PR TITLE
Add rubygem-unicode

### PR DESCRIPTION
**Pull request by ekohl**
This is a copy of the spec from EPEL7 with spec2scl applied and then
some manual changes to build inside SCL.

### CODE REVIEW

File: rubygem-unicode/license.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+../.git/annex/objects/q6/66/SHA256E-s2504--5572bf03c3c8c4770e8153038f5f54faad9c3db9900864a7c82a19de4434d74b.txt/SHA256E-s2504--5572bf03c3c8c4770e8153038f5f54faad9c3db9900864a7c82a19de4434d74b.txt
```
Comment (ehelms) [inline rubygem-unicode/license.txt]: I don't think setup_sources will pickup Source1 but this is such a small, non-binary file that you can include it directly.

File: rubygem-unicode/rubygem-unicode.spec [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+%if 0%{?rhel}
+ruby -I.%{gem_instdir}/lib:.%{gem_extdir_mri} test/test.rb
+%else
+ruby-mri -I.%{gem_instdir}/lib:.%{gem_extdir_mri} test/test.rb
+%endif
+%{?scl:EOF}
+
+
+%files
+%doc README license.txt
```
Comment (ehelms) [inline rubygem-unicode/rubygem-unicode.spec]: I'd split this into:

```
%doc README
%license license.txt
```

**Comment by ehelms** [comment]
You'll want to add this to the appropriate tag listings in rel-eng/tito.props as well

**Comment by ekohl** [comment]
@ehelms updated

**Comment by ekohl** [comment]
Running this on EL7 does cause hammer not to die but it shows:
```
Ignoring unicode-0.4.4.1 because its extensions are not built.  Try: gem pristine unicode --version 0.4.4.1
```

**Comment by ekohl** [comment]
It does open ```/opt/theforeman/tfm/root/usr/lib64/gems/ruby/unicode-0.4.4.1/unicode/unicode_native.so``` so I don't understand why it doesn't work.

**Comment by ekohl** [comment]
Updated, looks like the ```gem.build_complete``` file needs to exist.

**Pull request merged**